scripts/entrypoint.sh

```diff
@@ -58,6 +58,19 @@ wait_for_ipam() {
     return 1
 }
 
+echo -n "Copying portmap binary ... "
+
+HOST_PORTMAP="$HOST_CNI_BIN_PATH/portmap"
+if [[ -f "$HOST_PORTMAP" ]]
+then
+    mv "$HOST_PORTMAP" "$HOST_PORTMAP.old"
+    cp portmap "$HOST_CNI_BIN_PATH"
+    rm "$HOST_PORTMAP.old"
+else
+    cp portmap "$HOST_CNI_BIN_PATH"
+fi
```

Comment:
Having taken [another look](https://superuser.com/a/1139861/82514), I think this could be simplified even more. What do you think about:
```
HOST_PORTMAP="$HOST_CNI_BIN_PATH/portmap"
if [[ -f "$HOST_PORTMAP" ]]; then
    rm "$HOST_PORTMAP"
fi
cp portmap "$HOST_CNI_BIN_PATH"
```

Reply:
Agreed @mogren. ++

Reply:
Simplification engaged